common-arcana.lic

```diff
@@ -101,12 +101,14 @@ module DRCA
     case match
     when 'Your desire to prepare this offensive spell suddenly slips away'
       pause 1
-      return prepare?(abbrev, mana, symbiosis, command)
+      return prepare?(abbrev, mana, symbiosis, command, tattoo_tm)
     when 'Something in the area interferes with your spell preparations'
       DRC.bput('rel symb', 'You release the', 'But you haven\'t') if symbiosis
       return false
     end
 
+    DRC.bput("target", get_data('spells').prep_messages) if tattoo_tm
```

Comment:
❓ Does `You begin to weave mana lines` need to be added to `prep_messages`?

Reply:
No clue, I will check, figured it was a part of it.

Reply:
Nope, it's in there. Ty for the double check!